### ISSUE TITLE
Mouse click highlight circle is offset on secondary monitor

### ISSUE THREAD

**Issue by amedeux** [bfritscher/carnac#15]
Hi thanks for writing Carnac. This tool is amazing for recording tutorials.
I record CG tutorials on a 2 screen setup, and I use the secondary monitor as the recorded area, so that I won't be showing the windows menu bar while recording.
However, with such configuration the mouse click highlight circle is offset and I could not find a way to centre it.
See behaviour here:[https://youtu.be/eVQ8BlebHHw]( https://youtu.be/eVQ8BlebHHw)
Is this a known issue?

**Comment by bfritscher** [comment]
Yes, the mouse highlights uses an overlay initialized for the main screen. So if screen dpi between primary and secondary screen is different it might misalign. If you set the other screen as primary does it work? ( I know it does not solve the issue of hidding the windows bar.

**Comment by amedeux** [comment]
Thanks, it does work if it's set on primary (needs a reboot, first).  I've resorted to unlocking the windows bar,  and move it to the now secondary monitor. Still need to find a solution to the ALT+TAB menu appearing on primary as I work and I need to switch tabs. Thanks for your answer.
